### DISCUSSION TITLE
「GPT商店」哪些應用受歡迎？

### DISCUSSION THREAD

**Discussion by Eileen050824** [1101-GS4719-C/Python-Community-GS4719-C#1966] [General]
OpenAI今年一月推出了「GPT商店」，開發人員可以在這裡訂製自己的GPT，希望激勵開發者和創作者為平台提供更多新鮮有趣的內容，讓用戶能持續享有新體驗。
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/160737635/c3989abe-f956-4abb-ada9-b87a94194ce6)
根據SimilarWeb的新數據，最受歡迎的前五大類目是依序是「藝術與設計類」、「教育類」、「生產力類」、「寫作類」、「寫程式類」。我有親身使用過或較有名的例子像是Canva的GPT，能直接在ChatGPT中建立簡報、社群媒體貼文、Logo，提供更方便的線上圖像設計。其他我聽過的像是Grimoire的GPT，適合想打造網站但不知如何寫程式者。
然而有這麼多工具，看似能為我們的生活帶來極大轉變，但我認為最理想的時刻尚未到來。主要是Open AI沒有把搜尋列和介紹頁面做清楚，起初我其實也不太知道哪些工具可以協助我，也算是靠自己去熟悉和摸索。這讓我想到賈伯斯的名言：「在我們向客戶展示之前，客戶不知道他們想要什麼。」這句話或許驚人地適合現今缺乏引導的GPT商店。
在我看來，GPT商店距離走進人們的日常生活中仍有進步空間，也代表著一座成功的應用商店還需要一段時間醞釀...
資料來源：天下雜誌

**Comment by yyujimmy** [comment]
GPT商店的出現確實為開發人員和創作者提供了一個平台，讓他們可以創造出更多有趣和實用的應用。這些應用涵蓋了多個領域，從藝術和設計到教育、生產力和寫作等，為用戶提供了更多元化的體驗。

然而，你提到的一些問題確實值得關注。缺乏清晰的搜尋功能和介紹頁面可能會使得用戶難以找到他們需要的工具，這可能會影響到商店的使用率和效果。提供更好的導引和引導，幫助用戶找到合適的工具，確實是GPT商店需要改進的地方之一。

此外，賈伯斯的那句名言也提醒我們，有時候用戶可能不知道自己需要什麼，因此好的設計和介紹也能幫助用戶發現新的應用場景和可能性。因此，GPT商店需要更好地理解用戶的需求，提供更有價值的工具和服務，才能真正走進人們的日常生活中，實現其應有的價值。

**Comment by yubo0oo** [comment]
雖然有這麼多工具看似能為我們的生活帶來極大轉變，但我覺得最理想的時刻尚未到來。主要是OpenAI沒有把搜尋列和介紹頁面做得足夠清楚，一開始我也不知道哪些工具能協助我，都是靠自己去熟悉和摸索。這讓我想到賈伯斯的名言：「在我們向客戶展示之前，客戶不知道他們想要什麼。」這句話或許驚人地適合現今缺乏引導的GPT商店。

在我看來，GPT商店距離真正走進人們的日常生活中還有進步空間。這也意味著，要成為一個成功的應用商店，還需要更多時間來醞釀和完善。我很期待未來的發展，希望這個平台能更好地引導用戶，讓大家都能找到適合自己的工具，從而真正改變我們的生活
